azure/services/service.ftl

```diff
@@ -41,3 +41,4 @@
 
 [#-- Pseudo services --]
 [#assign AZURE_BASELINE_PSEUDO_SERVICE = "baseline"]
+[#assign AZURE_AAD_USERPOOL_PSEUDO_SERVICE = "microsoft.aad.userpool"]
```

Comment:
Is the service userpool? I would have thought it would be aad.appregistration?

Reply:
Yep, I agree in hindsight :)

Reply:
updated: 8586e25